api/v1beta1/openstackconfiggenerator_types.go

```diff
@@ -24,6 +24,8 @@ import (
 type OpenStackConfigGeneratorSpec struct {
 	// Name of the image used to generate configs. If missing will be set to the configured OPENSTACKCLIENT_IMAGE_URL_DEFAULT in the CSV for the OSP Director Operator.
 	ImageURL string `json:"imageURL,omitempty"`
+	// Optional. List of heat environment files to include from tripleo-heat-templates/environments
+	HeatEnvs []string `json:"heatEnvs,omitempty"`
```

Comment:
maybe default to an empty list like here https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/api/v1beta1/openstacknetattachment_types.go#L37

Reply:
will try it now...